File: GXpert/GXpert.Web/Modules/Content/ContentLicense/ContentLicenseColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace GXpert.Content.Columns;

[ColumnsScript("Content.ContentLicense")]
[BasedOnRow(typeof(ContentLicenseRow), CheckNames = true)]
public class ContentLicenseColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    [EditLink]
    public string Name { get; set; }
    public string Description { get; set; }
    public DateTime InsertDate { get; set; }
    public int InsertUserId { get; set; }
    public DateTime UpdateDate { get; set; }
    public int UpdateUserId { get; set; }
    public short IsActive { get; set; }
}

File: GXpert/GXpert.Web/Modules/ServerTypes/Content/ContentLicenseColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { ContentLicenseRow } from "./ContentLicenseRow";

export interface ContentLicenseColumns {
    Id: Column<ContentLicenseRow>;
    Name: Column<ContentLicenseRow>;
    Description: Column<ContentLicenseRow>;
    InsertDate: Column<ContentLicenseRow>;
    InsertUserId: Column<ContentLicenseRow>;
    UpdateDate: Column<ContentLicenseRow>;
    UpdateUserId: Column<ContentLicenseRow>;
    IsActive: Column<ContentLicenseRow>;
}

export class ContentLicenseColumns extends ColumnsBase<ContentLicenseRow> {
    static readonly columnsKey = 'Content.ContentLicense';
    static readonly Fields = fieldsProxy<ContentLicenseColumns>();
}